libvast/src/error.cpp

```diff
@@ -11,45 +11,94 @@
  * contained in the LICENSE file.                                             *
  ******************************************************************************/
 
+#include <sstream>
+
+#include <caf/pec.hpp>
+#include <caf/sec.hpp>
+
 #include "vast/detail/assert.hpp"
 #include "vast/error.hpp"
 
 namespace vast {
 namespace {
 
 const char* descriptions[] = {
-  "no_error",
+  "no error",
   "unspecified",
-  "no_such_file",
-  "filesystem_error",
-  "type_clash",
-  "unsupported_operator",
-  "parse_error",
-  "print_error",
-  "convert_error",
-  "invalid_query",
-  "format_error",
-  "end_of_input",
-  "version_error",
-  "syntax_error",
-  "invalid_table_slice_type",
-  "invalid_synopsis_type",
-  "remote_node_down",
-  "invalid_result",
-  "invalid_configuration",
-  "unrecognized_option",
-  "invalid_subcommand",
-  "missing_subcommand",
-  "no_importer",
+  "no such file",
```

Comment:
I'm not sure about the implications of removing the `_`s. In user-facing output it sure looks better, but this means parsing `inspect` output could become an issue in the future. I just want to bring it up, though, and leave the decision to you.

Reply:
I didn't know white space is troublesome for message serialization.